tronpy/proto/src/tron.proto

```
@@ -0,0 +1,153 @@
+syntax = "proto3";
```

Buf (1.54.0)

Files must have a package defined.

(PACKAGE_DEFINED)